docat/docat/utils.py

```diff
@@ -114,20 +114,23 @@ def get_all_projects(upload_folder_path: Path) -> Projects:
     Returns all projects in the upload folder.
     """
 
-    def has_not_hidden_versions(project):
+    def count_not_hidden_versions(project):
         path = upload_folder_path / project
         return any(
```

Comment:
shouldn't this be `len(...)` instead of `any(...)` ?

Reply:
oh yeah, I didn't see that. I'll change it...

Reply:
Ok, I fixed it and wrote a test for it.